src/index.css

```diff
@@ -1,3 +1,4 @@
+/* Left this code in case I need this file
```

Comment:
- [ Required ] I think it would be better if you delete this file from your code base since you are not using it to style any component and also all the code in the file is commented out. Kindly remove this file from your code base.